docs/adapters/development.md

```diff
@@ -33,7 +33,7 @@ class Sample extends Adapter {
     }
     run() {
         this.robot.logger.info('Run')
-        this.emit('connected')
+        this.emit('connected') // this line is required. Scripts won't get loadded if this event never fires.
```

Comment:
s/loadded/loaded/

For clarity, let's rephrase this to phrase positively instead of with a double negative, eg "The 'connected' event is required to trigger loading of Hubot scripts."